packages/platform/types/event-types/event-types_2024_06_14/outputs/event-type.output.ts

```diff
@@ -419,25 +420,37 @@ export class TeamEventTypeOutput_2024_06_14 extends BaseEventTypeOutput_2024_06_
   @DocsProperty()
   hideCalendarEventDetails?: boolean;
 
+  @IsObject()
+  @IsOptional()
+  @DocsProperty()
+  team!: Team | null;
```

Comment:
If you add docs property did you check the docs that the team object is actually appearing in docs? Because i think first need to add @ValidateNested decorator to actually validate the team, and then add `@Type(() => Team)`. Also, there might already be a dto called Team so better rename it to EventTypeTeam.

Reply:
After you do this and then rebuild platform types locally and run v2 and go to http://localhost:5555/docs and confirm team is seen in output its good to go.

Reply:
<img width="1512" alt="Screenshot 2024-12-17 at 3 28 21 PM" src="https://github.com/user-attachments/assets/6361f57b-3e31-4c38-bf39-d1cdc489a037" />